src/js/components/Notification/stories/typescript/Themed.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import { Box, Grommet, Notification, Text } from 'grommet';
+
+import { deepMerge } from 'grommet/utils';
+import { grommet, ThemeType } from 'grommet/themes';
+import { DisabledOutline } from 'grommet-icons';
+
+// Type annotations can only be used in TypeScript files.
+// Remove ': ThemeType' if you are not using Typescript.
+const customTheme: ThemeType = deepMerge(grommet, {
+  global: {
+    font: {
+      family: 'Arial',
+    },
+  },
+  notification: {
+    container: {
+      // any BoxProps
+      pad: { horizontal: 'large', vertical: '36px' },
+      background: '#aaccff',
+      round: 'small',
+    },
+    iconContainer: {
+      // any BoxProps
+      justify: 'center',
+      align: 'center',
+    },
+    textContainer: {
+      // any BoxProps
+      gap: '25px',
+    },
+    title: {
+      // any TextProps
+      color: '#0223df',
+    },
+    message: {
+      // any TextProps
+      color: {
+        light: '#0223df',
+        dark: '#6101a0',
+      },
+    },
+    close: {
+      icon: DisabledOutline,
+    },
+    undefined: {
+      color: '#b0ff7b',
+    },
+  },
+});
+
+export const Themed = () => (
+  <Grommet theme={customTheme}>
+    <Box pad="large" justify="center" gap="large">
+      <Box gap="xsmall">
+        <Text size="medium">Default (No status prop)</Text>
+        <Notification
```

Comment:
![Screen Shot 2021-08-19 at 11 50 02 AM](https://user-images.githubusercontent.com/1756948/130119646-7a93eb35-0e09-4f24-9919-7bfdf1023056.png)

Should the size of this be limited/contained? If so, where do you think the most appropriate place to constrain?

Colors tweaked? Worried about the lack of contrast between grey icon and blue background. Perhaps grab Aditi or Vicky for quick suggestions?

Reply:
for the overall notification component, there is no set size or limit. this is because right now we're thinking with banner notifications in mind, in which case the notification would span the width of whatever box it's contained in (probably the screen width) and height is based on the content